src/main/java/com/defano/wyldcard/part/field/styles/HyperCardTextPane.java

```diff
@@ -462,6 +463,15 @@ public void autoSelectLines(Set<Integer> selectedLines) {
             }
         }
     }
+    @RunOnDispatch
+    public void setDirty(boolean dirty) {
```

Comment:
Very minor, but `@RunOnDispatch` is unnecessary here. This is intended to denote that the method invokes a Swing API that is only allowed to run on the dispatch thread (Swing does not warn you if you do this; you just introduce very hard to debug problems, which is why I built this thread auditor thing-a-ma-bob.)

Reply:
Got it. Thanks!